include/grl/flatbuffer/KUKAiiwa.fbs

```diff
@@ -63,7 +63,9 @@ enum EControlMode : byte {
   POSITION_CONTROL_MODE=0,
   CART_IMP_CONTROL_MODE=1,
   JOINT_IMP_CONTROL_MODE=2,
-  NO_CONTROL=3
+  NO_CONTROL=3,
```

Comment:
I guess "CartesianSineImpedanceControlMode" is not currently available for FRI. I will revert this.